docs/source/commandline/commandline-all.rst

```diff
@@ -107,54 +144,76 @@ Note: By default, all the '**-list**' commands below write info to the screen. I
 		* 8: means some files were identified
 		* 9: means no files were identified
 
+.. _mame-commandline-listdevices:
+
 **-listdevices** / **-ld** [<*gamename|wildcard*>]
 
         Displays a list of all devices known to be hooked up to a game.  The ":" is considered the game itself with the devices list being attached to give the user a better understanding of what the emulation is using.
 
+.. _mame-commandline-listslots:
+
 **-listslots** [<*gamename|wildcard*>]
 
         Show available slots and options for each slot (if available).  Primarily used for MAME to allow control over internal plug-in cards, much like PCs needing video, sound and other expansion cards.
 		
         The slot name (e.g. **ctrl1**) can be used from the command line (**-ctrl1** in this case) 
 
+.. _mame-commandline-listmedia:
+
```

Comment:
I'm pretty sure this can't work with a wildcard.  It's another one where the fact that it updates when you change slot options is important, too.

Reply:
Like previous, does work with wildcards, leaving in.